README.md

```diff
@@ -39,27 +39,31 @@ const { Video, Data } = muxClient;
 As an example, you can create a Mux asset and playback ID by using the below functions on your Video instance.
 ```javascript
 // Create an asset
-let assetId;
-Video.assets.create({ input: 'https://storage.googleapis.com/muxdemofiles/mux-video-intro.mp4' })
-  .then((res) => {
-    const { data } = res.data;
-    assetId = data.id;
-  });
+const asset = await Video.assets.create({ input: 'https://storage.googleapis.com/muxdemofiles/mux-video-intro.mp4' });
```

Comment:
This is a great example, but I'm afraid it might scare away those who haven't yet learned the `await` syntax. Maybe we can provide both examples?

Reply:
Ah, never mind, I see you placed the promise example down below.